src/app/common/utils/system-notifications/system-notifications.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { IShowNotificationConfig } from './system.notifications.types';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SystemNotificationsService {
+    public constructor(
+        private readonly snackBar: MatSnackBar,
+      ) { }
+
+      public showNotification({ type, message, dismissBtnText = 'OK' }: IShowNotificationConfig): void {
```

Comment:
Please also run linter, there are some issues that need fixing including this one.